src/main/java/org/andersen/homework/util/collection/MyArrayList.java

```diff
@@ -0,0 +1,80 @@
+package org.andersen.homework.util.collection;
+
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class MyArrayList<E> {
+
+  private static final Object[] DEFAULT_EMPTY_CAPACITY_ARRAY = {};
+
+  private Object[] array;
+
+  private int capacity;
```

Comment:
Why do we need this field?

Reply:
This field contains amount of allocated memory for the 'array' field. Number of elements in an array may change, but it's capacity - never